apis/apps/v1alpha1/type.go

```diff
@@ -1133,6 +1137,27 @@ type ServiceRefVarSelector struct {
 	ServiceRefVars `json:",inline"`
 }
 
+// ComponentVarSelector selects a var from a Component.
+type ComponentVarSelector struct {
+	// The Component to select from.
+	ClusterObjectReference `json:",inline"`
+
+	ComponentVars `json:",inline"`
+}
+
+type ComponentVars struct {
+	// Reference to the replicas of the component.
+	//
+	// +optional
+	Replicas *VarOption `json:"replicas,omitempty"`
+
+	// Reference to the podName list of the component.
+	// and the value will be presented in the following format: podName1,podName2...
+	//
+	// +optional
+	PodNames *VarOption `json:"podNames,omitempty"`
```

Comment:
consider calling it `InstanceNames` ? as Instance has been defined as the fundmental unit of KB from the 4-layer model.

Reply:
> consider calling it `InstanceNames` ? as Instance has been defined as the fundmental unit of KB from the 4-layer model.

fixed